src/zcl_abapgit_repo_online.clas.testclasses.abap

```diff
@@ -1,4 +1,4 @@
-CLASS ltcl_code_inspector DEFINITION FOR TESTING.
+CLASS ltd_code_inspector DEFINITION FOR TESTING.
```

Comment:
please prefix test class name with "LTCL_", will enable https://github.com/larshp/abaplint/blob/master/src/rules/local_class_naming.ts sometime soon

Reply:
there was also a convention for mock classes, and also another one called something else? anyhow, suggest just calling it LTCL_whatever, in this case `LTCL_code_inspector_double` might work, can a double also be a mock? Prefixing with LTCL_ is easy for everyone

Reply:
Ok, convinced. ;-) 
Fixed.